### DISCUSSION TITLE
twitch

### DISCUSSION THREAD

**Discussion by Reodia** [ilyhalight/voice-over-translation#498] [Q&A]
Браузер: Firefox Browser 121.0.1 (64-разрядный)
Плагин: 1.5.0.5

В описании указано, что перевод на твиче поддерживается, но, к сожалению, не смог найти кнопку перевода ни на стримах онлайн, ни на записях(/videos), ни в клипах, мобильная версия так же тестировалась.
У кого-то данный функционал работает? И если да - в какой конфигурации?

**Comment by NullVerdict** [comment]
> Браузер: Firefox Browser 121.0.1 (64-разрядный) Плагин: 1.5.0.5
> 
> В описании указано, что перевод на твиче поддерживается, но, к сожалению, не смог найти кнопку перевода ни на стримах онлайн, ни на записях(/videos), ни в клипах, мобильная версия так же тестировалась. У кого-то данный функционал работает? И если да - в какой конфигурации?

Зайди на клип или видео и обнови страницу

**Reply by Reodia** [reply]
Не один раз, с очисткой кеша, в инкогнито.

**Reply by NullVerdict** [reply]
Попробуй тогда установить скрипт из [моего репозитория](https://github.com/SashaXser/voice-over-translation/tree/dev/dist)

**Reply by NullVerdict** [reply]
Есть прогресс?
